README.md

```diff
@@ -1 +1,47 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+This project demostrates usage of linters in html and css
+
+# Built With
+-Html and css
+
+# Getting Started
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+
+# Prerequisites
+-text editor
+-nodejs installed on your local machine
+
+# Setup
+-Clone the repository
+-Open the project in your favourite text editor
+
+# Install
+-Run npm install to install project dependencies
+
+# Authors
+-GitHub: https://github.com/Daniel-Kigozi,
+-Twitter: @DanielKigozi15,
+-LinkedIn: https://www.linkedin.com/in/daniel-kigozi-375182208/
+
```

Comment:
**[OPTIONAL]:** Instead of adding the full URL to your accounts, you might want to consider using markup like this:

`Github: [@yourhandle](URL of your account/profile)`

And the final output will look like this (with your own information):
![image](https://user-images.githubusercontent.com/92395484/191323940-51381d31-b872-4ada-8c13-52b413310b12.png)

This looks cleaner and will make it easy for anyone who visits your repo to access your profiles :sunglasses: :heavy_check_mark: